.gitignore

```diff
@@ -1,11 +1,24 @@
-/.bundle/
+# Ignore all logfiles and tempfiles
+/tmp/
+/tags
+
+# Environment normalisation
+/.rspec-local
 /.yardoc
-/_yardoc/
-/coverage/
-/doc/
+/_yardoc
+/lib/bundler/man/
+
+/docs/
```

Comment:
What is the reason to ignore [`/docs`](https://github.com/babbel/puma-plugin-telemetry/tree/main/docs)?

Reply:
also doc is not not gitignored 🤔

Reply:
Honestly, that was just a miss. My fault, I'll remove it.